app/models/data_model/publication.rb

```diff
@@ -0,0 +1,7 @@
+module DataModel
+	class Publication < ActiveRecord::Base
+		include Genome::Extensions::UUIDPrimaryKey
+		has_and_belongs_to_many :interactions
```

Comment:
ok I wasn't sure about this. From this I thought I needed to put it in both models http://guides.rubyonrails.org/association_basics.html#the-has-and-belongs-to-many-association